style.css

```diff
@@ -448,3 +489,463 @@ footer .contact-para {
   margin-top: 18px;
   font-size: 18px;
 }
+
+@media screen and (min-width: 992px) and (max-width: 1100px) {
+  .first-page {
+    border-bottom-left-radius: 150px;
+    width: 100%;
+    padding-top: 20px;
+    height: 900px;
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: center;
+    margin: 0 auto;
+    background-color: white;
+    background-image:
+      url('./images/dhead1.svg'),
+      url('./images/dhead2.svg'),
+      url('./images/dhead3.svg'),
+      url('./images/dhead4.svg'),
+      url('./images/dhead5.svg'),
+      url('./images/dhead6.svg'),
+      url('./images/dhead7.svg'),
+      url('./images/dhead8.svg'),
+      url('./images/dhead9.svg'),
+      url('./images/dhead10.svg');
+    background-position:
+      98% 21%,
+      89% 11%,
+      48% 21%,
+      8% 22%,
+      3% 17%,
+      63% 82%,
+      58% 85%,
+      18% 80%,
+      17% 83.5%,
+      48% 23%;
+    background-repeat: no-repeat;
+  }
+
+  li {
+    list-style-type: none;
+    justify-content: space-between;
+  }
+
+  #head .menu-link {
+    display: flex;
+    padding-right: 60px;
+  }
+
+  .icons li {
+    list-style-type: none;
+    margin-right: 40px;
+    margin-left: -20px;
+  }
+
+  #head .menu-link li {
+    padding-right: 30px;
+    text-decoration: none;
+  }
+
+  .picture img {
+    width: 400px;
+    margin: 0 auto;
+    padding: 0;
+  }
+
+  .javascript img {
+    padding-top: 5px;
+    height: 55px;
+    width: 55px;
+  }
+
+  .html img {
+    padding-top: 5px;
+    height: 55px;
+    width: 55px;
+  }
+
+  .css img {
+    padding-top: 5px;
+    height: 55px;
+    width: 55px;
+  }
+
+  #head img {
+    display: none;
+  }
+
+  #head a {
+    margin-left: 15px;
+    font-weight: bold;
+    color: #344563;
+    text-decoration: none;
+  }
+
+  #head .logo {
+    color: #6070ff;
+    padding-left: 70px;
+  }
+
+  #head {
+    height: 40px;
+    display: flex;
+    justify-content: space-between;
+    margin: 0 auto;
+    align-items: center;
+    width: 100%;
+  }
+
+  /* header-main-section */
+
+  .headline {
+    height: 400px;
+    width: 700px;
+    margin: 0 auto;
+  }
+
+  .first-page .name {
+    font-size: 20px;
+    font-weight: 900;
+  }
+
+  .first-page .name h1 {
+    margin: 0;
+  }
```

Comment:
- [ ] Your desktop version looks good 👍, great job. I would recommend changing the size of fonts,  images, and buttons in the desktop version so your desktop version is complete. You can use the Figma design as a reference.

Reply:
Hi @safar1212, The requested changes are required. You must stick to the design as much as possible (font sizes, image sizes ... ) using the Figma Template provided.

Reply:
Thank you, sir, I modified them all.